HW_module_2.py

```diff
@@ -0,0 +1,54 @@
+import string
+import random
+
+# create a list of random number of dicts (from 2 to 10)
+
+dict_List = []                                             # empty list
+for x in range(random.randint(2, 10)):                     # random number of dictionaries
+#    size = random.randint(1,26)                          # random dictionary size
```

Comment:
Why have you chosen size 3 in the final version? size = random.randint(1,26) looks ok.
If you wanted to expand the length you could use size = random.randint(5,26) for example.